train.gemspec

```diff
@@ -19,11 +19,11 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '>= 2.0'
+  spec.required_ruby_version = '>= 2.3'
```

Comment:
I recommend that this change be omitted from this PR and performed on a separate PR related to removing old Ruby versions which we suspect requires a major version bump.